src/main.js

```diff
@@ -37,8 +38,19 @@ module.exports = {
     const branch = config.default ? await this._getDefaultBranch(gitInfo.githubUrl, vscodeConfig) : config.perma ? this._getGitLongHash(cwd) : gitInfo.branch
 
     const subdir = editor.document.fileName.substring(cwd.length)
-    let url = `${gitInfo.githubUrl}/blob/${branch}${subdir}#${lineQuery}`
-    url = url.replace(/\\/g, '/') // Flip subdir slashes on Windows
+
+    // Determine the path separator by using platform.
+    // Note: The config.platform overrides process.platform for easy to test.
+    let pathSeparator = ''
```

Comment:
Let's use path.sep and config.pathSeparator to be more precise here.  For example:
`let pathSeparator = config.pathSeparator || path.sep`

Reply:
Thank you for your review! It seems good to use pathSeparator rather than using platform. I fixed them.
https://github.com/differentmatt/vscode-copy-github-url/pull/33/commits/c43156e3b5f0e6cfc95ca1cb8de2f50d84eaa44e